package.json

```
@@ -105,10 +104,13 @@
   "main": "lib/index.js",
   "oclif": {
     "commands": "./lib/commands",
+    "hooks": {
+      "command_not_found": [
+        "./lib/hooks/command_not_found/example"
```

```suggestion
        "./lib/src/errors/command-not-found"
```
I think this should be the path of the hook, not really sure, looking at this example it looks like that

---

@Souvikns i've added the hook in `hooks/command-not-found` folder, it should work right ?

---

I think that should be a file path, check that after building the project does that path has any file

---

Yeah, it's just file paths that we have to link back in `package.json`. It's working build.

I just separated the hooks into folders based on events for further clarification if anyone wants to add new hooks.